File: 2469-convert-the-temperature/2469-convert-the-temperature.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
double* convertTemperature(double celsius, int* returnSize) {
    double* m = (double*)malloc(2 * sizeof(double));
    m[0] = celsius+273.15;
    m[1] = celsius * 1.80 + 32.00;
    *returnSize = 2;
    return m;
}

File: 2469-convert-the-temperature/2469-convert-the-temperature.py
class Solution:
    def convertTemperature(self, celsius: float) -> List[float]:
        m=[]
        Kelvin=Fahrenheit=0
        Kelvin = celsius + 273.15
        Fahrenheit = celsius * 1.80 + 32.00
        m.append(Kelvin)
        m.append(Fahrenheit)
        return m